config.yaml

```diff
@@ -26,12 +26,12 @@ options:
 
           This can be used to lock down access to Mattermost based on source IP address.
         default: ''
-    licence:
+    license:
```

Comment:
Is this a breaking change since the configuration option is being renamed? And what is the reason for the rename?

Reply:
This is what Google thinks
![Screenshot 2023-03-06 at 5 25 01 pm](https://user-images.githubusercontent.com/51036209/223034583-bb957178-788b-448c-9804-f6991307809a.jpg)

Reply:
Apparently it's only in the USA that license is noun and verb. I didn't know that.  
I thought that licence wasn't correct in english.  

I knew that this was a breaking change (as stated in the PR description).  

Knowing this, I think it's best to revert.

Reply:
Reverted